services/EventTokenService.js

```diff
@@ -0,0 +1,68 @@
+const jwt = require("jsonwebtoken");
+const crypto = require("crypto");
+
+// A service class for Token generation and management
+
+class EventTokenService {
+  static #app_access_key = config.get("Event100ms.APP_ACCESS_KEY");
+  static #app_secret = config.get("Event100ms.APP_SECRET");
+  #managementToken;
+  constructor() {
+    this.#managementToken = this.getManagementToken(true);
+  }
+
+  // A private method that uses JWT to sign the payload with APP_SECRET
+  #signPayloadToToken(payload) {
+    const token = jwt.sign(payload, EventTokenService.#app_secret, {
+      algorithm: "HS256",
+      expiresIn: "24h",
+      jwtid: crypto.randomUUID(),
```

Comment:
Take a look here and try disabling cache to ensure uniqueness
https://nodejs.org/api/crypto.html#cryptorandomuuidoptions

Reply:
Yes, added it. Thank you